src/layer/LayersFactory.ts

```diff
@@ -9,12 +9,16 @@ import {
   DATASET_MODIS,
   DATASET_AWS_DEM,
   DATASET_AWS_S1GRD_IW,
+  DATASET_S3SLSTR,
+  DATASET_S3OLCI,
```

Comment:
Should the `DATASET_AWS_S1GRD_EW` and `S3SLSTRLayer` be imported as well?

Reply:
As we discussed, I had a branch ready that joined Sentinel 1 IW and EW in a single layer, which I have now merged to this branch. The reason was that apart from `AcquisitionMode`, there are also `Polarization` and `Resolution` parameters which also break dataset into partitions - it would not be consistent to only have `Layer` classes only for `AcquisitionMode`, and having all possible combinations would not be viable.

Reply:
Resolving.